src/util/dwarf.rs

```diff
@@ -150,18 +150,34 @@ impl FundType {
             FundType::Vec2x32Float => "__vec2x32float__",
         })
     }
+
+    pub fn parse_int(value: u16) -> Result<Self, TryFromPrimitiveError<Self>> {
+        if value >> 8 == 0x1 {
+            // Can appear in erased tags
+            Self::try_from(value & 0xFF)
+        } else {
+            Self::try_from(value)
+        }
+    }
 }
 
 #[derive(Debug, Eq, PartialEq, Copy, Clone, IntoPrimitive, TryFromPrimitive)]
 #[repr(u8)]
 pub enum Modifier {
+    MwPointerTo = 0x00, // Used in erased tags
```

Comment:
How does this happen?

Reply:
There are 3 values we need to handle in the OOT elf:
* `00`: used for all pointers except `signed int` (not sure what to call this)
* `80`: used for pointers to `signed int` (maybe should be called `MwPointerTo`?)
* `82`: not sure, it's only used twice, for a local variable `signed int & pDL` in two different functions (maybe should be called `MwReferenceTo`? I'm still confused about pointer vs reference here)
```